src/value/index.ts

```diff
@@ -61,6 +61,13 @@ export class MotionValue<V = any> {
      */
     private updateSubscribers = subscriptionManager<Subscriber<V>>()
 
+    /**
+     * Functions to notify when the velocity updates.
+     *
+     * @internal
+     */
+    public velocityUpdateSubscribers = subscriptionManager<Subscriber<number>>()
```

Comment:
Why do we need a separate set of velocitySubscribers? Is that an optimisation because the velocity does not change as much?

Reply:
Hopefully explained in the other comment